packages/angular_devkit/build_angular/src/angular-cli-files/models/webpack-configs/common.ts

```diff
@@ -254,6 +254,20 @@ export function getCommonConfig(wco: WebpackConfigOptions): Configuration {
   }
 
   if (scriptsOptimization) {
+    let angularGlobalDefinitions = {
+      ngDevMode: false,
+      ngI18nClosureMode: false,
+    };
+
+    try {
+      // Try to load known global definitions from @angular/compiler-cli.
+      // tslint:disable-next-line:no-implicit-dependencies
+      const GLOBAL_DEFS_FOR_TERSER = require('@angular/compiler-cli').GLOBAL_DEFS_FOR_TERSER;
+      angularGlobalDefinitions = GLOBAL_DEFS_FOR_TERSER;
```

Comment:
If `GLOBAL_DEFS_FOR_TERSER` doesn't exist, won't this wipe out the defaults?  I think this should probably check for undefined and only set if truthy.

Reply:
Yes, that's absolutely right. Will change.